### PR TITLE
Fix issue with dapp suggested option being visible in edit gas fee popover even if dapp has no gas suggestion

### LINKED ISSUE #13312: [Bug]: EIP-1559 v2 Site Suggested gas fee bugs
### Describe the bug

Bugs that @jakehaugen uncovered during UI acceptance testing.

These two bugs pertain to "site suggested" gas fees. The latest build linked on [Slack](https://consensys.slack.com/archives/C021UN08FBJ/p1642010438018600) was used.

### Steps to reproduce

1. The site suggested price does not appear to be selected by default (at least on UniSwap, still going to “Market”).
2. The estimate in the “edit” dialog is showing 0 for everything. So, something not making it into the dialog.

image (2)

### Error messages or log output

_No response_

### Version

10.8.2

### Build type

Other (please specify exactly where you obtained this build in "Additional Context" section)

### Browser

Chrome

### Operating system

MacOS

### Hardware wallet

_No response_

### Additional context

Test build

### PR DESCRIPTION
**Pull request by jpuri**
Fixes: #13312

Explanation:  Dapp-Suggested option should not be visible in edit gas fee modal if dapp has not suggested any gas fee.

Screenshot 2022-01-14 at 1 04 53 PM
 
The option was hidden if there was no dapp suggestions, but I found scenario with uniswap where dapp is suggesting gasLimit but not price. Thus I have changed condition to check dapp suggested gas price to look for maxFeeParGas / gasPrice specifically.

### CODE REVIEW

**Review by adonesky1 — DISMISSED** [review]
LGTM

**Review by ryanml — DISMISSED** [review]

File: ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js [hunk 1]
```diff
@@ -64,7 +64,8 @@ const EditGasItem = ({ priorityLevel }) => {
 
   if (
     priorityLevel === PRIORITY_LEVELS.DAPP_SUGGESTED &&
-    !dappSuggestedGasFees
```
Comment (mcmire) [inline ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js:67]: What do you think about adding a test for when `dappSuggestedGasFees` is present, but neither `dappSuggestedGasFees.maxFeePerGas` nor `dappSuggestedGasFees.gasPrice` is present?

Comment (jpuri) [reply ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js:67]: added 👍

**Review by adonesky1 — APPROVED** [review]
👍

**Review by darkwing — APPROVED** [review]

**Pull request merged**